src/components/box/Box.spec.jsx

```diff
@@ -29,128 +28,82 @@ test('border', () => {
   const box = shallow(boxComponent);
 
   expect(boxComponent.props.border).toEqual(true);
-  expect(box.hasClass('sg-box--no-border')).toEqual(false);
+  expect(box.hasClass('sg-box--with-border')).toEqual(true);
```

Comment:
maybe we could get rid of `with`, wdyt? just `--border`

Reply:
@matzimowski https://github.com/brainly/style-guide/pull/1864/commits/5b81d331777a0660cb07981de44408846a89ee50